src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -571,7 +571,7 @@ enum OpcodeClient : uint32
     CMSG_REQUEST_HONOR_STATS                          = 0x1207,
     CMSG_REQUEST_LFG_LIST_BLACKLIST                   = 0x0A02,
     CMSG_REQUEST_PARTY_JOIN_UPDATES                   = 0x0620,
-    CMSG_REQUEST_PARTY_MEMBER_STATS                   = 0x0EB8,
+    CMSG_REQUEST_PARTY_MEMBER_STATE                   = 0x0EB8,
```

Comment:
**NO** renaming opcodes - the names currently present here are final (except unknowns)